src/core/methods/trackTransactions/trackTransactions.ts

```diff
@@ -1,44 +1,34 @@
-import { getTransactionsByHashes as defaultGetTxByHash } from 'apiCalls/transactions/getTransactionsByHashes';
+import { getTransactionsByHashes } from 'apiCalls/transactions/getTransactionsByHashes';
 import { websocketEventSelector } from 'store/selectors/accountSelectors';
 import { getStore } from 'store/store';
 import { checkTransactionStatus } from './helpers/checkTransactionStatus';
 import { getPollingInterval } from './helpers/getPollingInterval';
-import { TransactionsTrackerType } from './trackTransactions.types';
 import {
   websocketConnection,
   WebsocketConnectionStatusEnum
 } from '../initApp/websocket/websocket.constants';
 
 /**
  * Tracks transactions using websocket or polling
- * @param props - optional object with additional websocket parameters
  * @returns cleanup function
  */
-export async function trackTransactions(props?: TransactionsTrackerType) {
+export async function trackTransactions() {
   const store = getStore();
   const pollingInterval = getPollingInterval();
   // eslint-disable-next-line no-undef
   let pollingIntervalTimer: NodeJS.Timeout | null = null;
   let timestamp = websocketEventSelector(store.getState())?.timestamp;
 
-  // Check if websocket is completed
   const isWebsocketCompleted =
     websocketConnection.status === WebsocketConnectionStatusEnum.COMPLETED;
 
-  // Assign getTransactionsByHash based on props or use default
-  const getTransactionsByHash =
-    props?.getTransactionsByHash ?? defaultGetTxByHash;
-
-  // Function that handles message (checking transaction status)
   const recheckStatus = () => {
     checkTransactionStatus({
       shouldRefreshBalance: isWebsocketCompleted,
-      getTransactionsByHash,
-      ...props
+      getTransactionsByHash: getTransactionsByHashes
```

Comment:
strange name difference. & why not use it as a default in checkTransactionStatus and allow optional override

Reply:
removed it and used as default in checkTransactionStatus